comictaggerlib/renamewindow.py

```diff
@@ -81,7 +83,14 @@ def config_renamer(self, ca: ComicArchive, md: GenericMetadata | None = None) ->
             new_ext = ca.extension()
 
         if md is None:
-            md = ca.read_metadata(self.data_style)
+            # TODO Same is used for taggerwindow.py:~1734/autotagmatchwindow Make a method?
+            md = GenericMetadata()
+            try:
+                for style in self.data_styles:
+                    md.overlay(ca.read_metadata(style))
+            except Exception as e:
+                logger.error("Failed to load metadata for %s: %s", ca.path, e)
+
```

Comment:
This bit of code is repeated in the mentioned windows. Is it worth getting them to use the same method?

Reply:
Since we are adding in multiple read styles yes